test/fixtures/controller.line/borderColor/scriptable.js

```diff
@@ -6,18 +6,17 @@ module.exports = {
 			datasets: [
 				{
 					// option in dataset
-					data: [0, 5, 10, null, -10, -5],
-					pointBorderColor: function(ctx) {
-						var value = ctx.dataset.data[ctx.dataIndex] || 0;
-						return value > 8 ? '#ff0000'
-							: value > 0 ? '#00ff00'
-							: value > -8 ? '#0000ff'
+					data: [4, 5, 10, null, -10, -5],
+					borderColor: function(ctx) {
+						var index = (ctx.dataIndex === undefined ? ctx.datasetIndex : ctx.dataIndex);
+						return index === 0 ? '#ff0000'
+							: index === 1 ? '#00ff00'
 							: '#ff00ff';
```

Comment:
I would increase the `borderWidth` to make the test more robust but also make more obvious the point borders (maybe increase a little bit the default point radius as well (~16px?). I would pick `#0000ff`  instead of `#ff00ff` since there is only 3 colors.

Reply:
When I look at the expected PNG, the points border doesn't seem to change for the first dataset but I think it should since it's supposed to [fallback to the dataset `borderColor`](/chartjs/Chart.js/blob/master/src/controllers/controller.line.js#L168) if `pointBorderColor` is undefined.